apps/laboratory/tests/cloud-auth-email.spec.ts

```diff
@@ -55,6 +55,12 @@ cloudAuthEmailTest('should get session account', async () => {
   await modalValidator.expectSessionAccount()
 })
 
+cloudAuthEmailTest('should switch namespace', async () => {
+  await modalPage.switchNetwork('Solana')
+  await modalPage.closeModal()
+  await modalValidator.expectSession()
+})
```

Comment:
The session it finds here is probably evm session as solana does not have 1ca enabled